src/lib/Libifl/pbs_loadconf.c

```diff
@@ -255,6 +257,58 @@ parse_config_line(FILE *fp, char **key, char **val)
 	return ret;
 }
 
+
+/**
+ * @brief
+ * parse_psi - parses the PBS_SERVER_INSTANCES
+ *    
+ * @param[in] conf_value  configuration value set in pbs.conf
+ * 
+ * @return int
+ * @retval -1, For error
+ * @retval 0, For success
+ */
+
+int
+parse_psi(char *conf_value)
+{
+	char **list;
+	int i;
+
+	free(pbs_conf.psi);
+
+	list = break_comma_list(conf_value);
+	if (list == NULL)
+		return -1;
+
+	for (i = 0; list[i] != NULL; i++)
+		;
+
+	if (!(pbs_conf.psi = calloc(i, sizeof(pbs_server_instance)))) {
+		fprintf(stderr, "Ran out of memory parsing configuration %s", conf_value);
+		return -1;
+	}
+
+	for (i = 0; list[i] != NULL; i++) {
+		if (parse_pbs_name_port(list[i], pbs_conf.psi[i].name, &pbs_conf.psi[i].port) != 0) {
+			fprintf(stderr, "Error parsing PBS_SERVER_INSTANCES\n");
+			return -1;
+		}
+		if (pbs_conf.psi[i].name[0] == '\0')
+			strcpy(pbs_conf.psi[i].name, pbs_conf.pbs_server_name);
+		if (pbs_conf.psi[i].port == 0) {
+			if (strcmp(pbs_conf.psi[i].name, pbs_conf.pbs_server_name) == 0)
```

Comment:
I would say use `is_same_host` here, which take cares of checking short name vs fqdn

Reply:
I agree, addressed it.